activiti-spring-conformance-tests/activiti-spring-conformance-set2/src/test/java/org/activiti/spring/conformance/set2/UserTaskAssigneeDeleteRuntimeTest.java

```diff
@@ -102,7 +102,7 @@ public void shouldFailOnDeleteTask() {
                         TaskRuntimeEvent.TaskEvents.TASK_CREATED,
                         TaskRuntimeEvent.TaskEvents.TASK_ASSIGNED);
 
-        collectedEvents.clear();
+        cleanUp();
```

Comment:
Same here, I'd prefer to not reuse a method annotated with `@Before` inside a test.